packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/index.md

```diff
@@ -109,6 +109,65 @@ For password-only authentication, you need to update the password authenticator
 1. In the **Edit Rule** dialog box, select **Not required** in the **AND Additional verification is** section.
 1. Click **Update Rule**.
 
+### Set up your Okta org for a password-optional use case
+
+Set up your Okta your org to enable authentication without a password using the followings steps:
+
+1. [Set up the email authenticator](#set-up-the-email-authenticator)
+1. [Create a password-optional authenticator policy](#create-a-password-optional-authenticator-policy)
+1. [Add a new global session policy](#add-a-new-global-session-policy)
+1. [Add a new authentication policy](#add-a-new-authentication-policy)
+
+#### Set up the email authenticator
+
+1. In the Admin Console, go to **Security** > **Authenticator**.
+1. On the **Setup** tab, locate the **Email** authenticator and select **Actions > Edit**.
+1. Set **This authenticator can be used for** to **Authentication and recovery**.
+1. Click **Save**.
+
+#### Create a password-optional authenticator policy
+
+1. In the Admin Console, go to **Security** > **Authenticator**.
+1. Select the **Enrollment** tab, and then click **Add Multifactor Policy**.
+1. Give the policy a name.
+1. Set **Assign to groups** to a group (for example "Everyone").
+1. In the **Eligible Authenticators** section
+   1. Set **Email** to **Required**.
+   1. Set **Password** to **Optional**.
+   1. Ensure the remaining authenticators are set to **Optional**.
```

Comment:
```suggestion
   * Ensure that the remaining authenticators are set to **Optional**.
```

Reply:
Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435